8_wyznaczanie_pierwiastkow_wielomianow/8_4_2_bairstow.tex

```diff
@@ -44,21 +44,21 @@ \subsection{Pierwiastki zespolone -- metoda Bairstowa}
 \begin{frame}
   Pochodne znajdujemy w następujący sposób:
 
-  $P(x)$ -- nie zależy od $p$, $q$, \\ z (\ref{bairstow}) mamy:
+  $P(x)$ -- nie zależy od $p$, $q$, \\ Z równania (\ref{bairstow}) mamy:
 
   $$\left. \begin{array}{l}
-  0 = (x^2 + p \cdot x + q) \cdot \frac{{\partial}Q}{{\partial}q} + Q + \frac{{\partial}R}{{\partial}q} + \frac{{\partial}S}{{\partial}q} \\ % NIE POWINNO BYĆ *x PRZY R'???
-  0 = (x^2 + p \cdot x + q) \cdot \frac{{\partial}Q}{{\partial}p} + x \cdot Q + \frac{{\partial}R}{{\partial}p} + \frac{{\partial}S}{{\partial}p} % NIE POWINNO BYĆ *x PRZY R'???
+  0 = (x^2 + p \cdot x + q) \cdot \frac{{\partial}Q}{{\partial}q} + Q + x \cdot \frac{{\partial}R}{{\partial}q} + \frac{{\partial}S}{{\partial}q} \\
```

Comment:
Tu tylko słowem wyjaśnienia: przepisywaliśmy to z wcześniejszych dokumentów, w których były te (oby) błędy. Postanowiliśmy trzymać się w tym przypadku zasady "po pierwsze nie szkodzić" i nie poprawiać tego na własną rękę, gdyż bardzo głupio byłoby nam wprowadzić własne błędy,  a czasem można naprawdę się zdziwić. Mam nadzieję, że nasze wątpliwości zostały zweryfikowane ;)

Reply:
Spokojnie, weryfikacja się odbyła.